front-end/src/components/User.js

```diff
@@ -86,8 +84,10 @@ const User = ({ userId }) => {
 };
 
 User.propTypes = {
-  // eslint-disable-next-line react/require-default-props
+  // eslint-disable-next-line react/require-default-props,react/no-unused-prop-types
```

Comment:
My linter was complaining about `userId` not being used which is confusing since it is used...... And I didn't change anything here with `userId`, so it's confusing. Oh well